livesync/indico_livesync/handler.py

```diff
@@ -111,6 +115,9 @@ def _moved(obj, old_parent, **kwargs):
 def _created(obj, **kwargs):
     if not isinstance(obj, (Event, EventNote, Attachment, Contribution, SubContribution)):
         raise TypeError(f'Unexpected object: {type(obj).__name__}')
+    # we don't care about unlisted events
+    if isinstance(obj, Event) and not obj.category:
```

Comment:
This is not enough; you're still creating queue entries when adding contributions etc. within an unlisted event.